Neo.Storage.RocksDB/RocksDBCore.cs

```diff
@@ -0,0 +1,164 @@
+﻿using RocksDbSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+namespace Neo.Plugins.Storage
+{
+    public class RocksDBCore : IDisposable

```

Comment:
I don't quite understand this naming. What do you mean by RocksDBCore? It would be good to use the same naming scheme for all storage implementations. Maybe a class diagram could help us.

Reply:
Done.